packages/sdk/src/client_api/index.ts

```diff
@@ -153,11 +161,11 @@ export class DbConnection extends DbConnectionImpl<
     ErrorContext,
     SubscriptionEventContext
   > => {
-    return new DbConnectionBuilder<
-      DbConnection,
+    return new DBConnectionBuilder<
+      DBConnection,
       ErrorContext,
       SubscriptionEventContext
-    >(REMOTE_MODULE, (imp: DbConnectionImpl) => imp as DbConnection);
+    >(REMOTE_MODULE, (imp: DBConnectionImpl) => imp as DBConnection);
```

Comment:
Looks like some casing got reverted here somehow. This should be `DbConnectionImpl`.